### ISSUE TITLE
Training set and test set of real data

### ISSUE THREAD

**Issue by kyeongbokong** [jacenfox/ldr2hdr-public#1]
Hello, I download your dataset and I want to test your network using real dataset.

However, Real dataset dose not divide traning/test. Could you explain about this?

Thanks

**Comment by jacenfox** [comment]
The provided real dataset contains all the ldr/hdr pairs, you can decide how to split them. :)
